spring-boot/src/main/java/org/dcsa/conformance/springboot/ConformanceApplication.java

```diff
@@ -202,17 +202,15 @@ public ConformanceApplication(ConformanceConfiguration conformanceConfiguration)
 
   private void _addSimulatedLambdaDelay() {
     if (conformanceConfiguration.simulatedLambdaDelay > 0) {
-      log.info(
-          "Simulating lambda delay of %d milliseconds"
-              .formatted(conformanceConfiguration.simulatedLambdaDelay));
+      log.info("Simulating lambda delay of {} milliseconds", conformanceConfiguration.simulatedLambdaDelay);
```

Comment:
FYI: Using `{}` causes the logging framework to only use the given variable if the log level requires it. So, preventing formatting the String unnecessary.